.github/workflows/phpunit-tests.yml

```diff
@@ -154,13 +154,35 @@ jobs:
       phpunit-config: ${{ matrix.multisite && 'tests/phpunit/multisite.xml' || 'phpunit.xml.dist' }}
       report: ${{ matrix.report || false }}
 
+  #
+  # Runs specific individual test groups.
+  #
+  specific-test-groups:
+    name: ${{ matrix.phpunit-test-groups }}
+    uses: ./.github/workflows/reusable-phpunit-tests-v3.yml
+    permissions:
+      contents: read
+    secrets: inherit
+    if: ${{ github.repository == 'WordPress/wordpress-develop' || github.event_name == 'pull_request' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [ '8.4' ]
```

Comment:
I think it might be valuable to include some additional PHP versions here as well.  I don't think we necessarily need the full list that we do elsewhere, but would would you think of us doing for each major version of PHP, we include the earliest and latest supported minor.  So this would mean 7.2, 7.4, 8.0 and 8.4 right now.

Reply:
That works for me. Updated!

Reply:
I'm not opposed, but I'm less concerned about the range of PHP versions here because there are unit tests covering HTML API, and I'd expect those to catch problems with version incompatibility. This suite is about the tree structure logic of the HTML Processor. It's still susceptible to version differences, but I wouldn't expect anything unique to appear in this suite.